### ISSUE TITLE
Operator>>, <<, -> overloading bug

### ISSUE THREAD

**Issue by Hyrae** [svenevs/exhale#168]
When parsing some overloaded operators, exhale misinterprets it for a template and produces an error.
```
(!) Invalid C++ template: operator<< has 2 '<' and 0 '>', exhale does not know what to do with your code.NoneType: None
```
Minimal example:
```c++
std::ostream& operator<<(std::ostream& os, const MyType& t) {
    os << t.toString();
    return os;
}
```
This happens with operators `<<`, `>>` and `->`.

**Comment by Hyrae** [comment]
There should be a test here if the node contains the `operator` name:

https://github.com/svenevs/exhale/blob/c10eddd72f64f0e5a3865e3d59b76043bbed75f3/exhale/graph.py#L2434-L2439

**Comment by GamesTrap** [comment]
In my project this also happens with `operator<`, `operator>`, `operator<=`, `operator>=`

`(!) Invalid C++ template: TRAP::Network::operator< has 1 '<' and 0 '>', exhale does not know what to do with your code.NoneType: None`

**Comment by svenevs** [comment]
Whoops, thanks for reporting!

Fix inbound when I'm available, should be a straightforward fix. This is a regression from the 0.3.2 these, use 0.3.1 if you need docs building ASAP but I'll have a new release out this weekend fixing it.

**Comment by svenevs** [comment]
Fixed in [v0.3.4](https://pypi.org/project/exhale/0.3.4/), you should update your requirements to need `exhale>=0.3.4`.
